jkube-kit/enricher/specific/src/test/java/org/eclipse/jkube/kit/enricher/specific/AbstractHealthCheckEnricherTest.java

```diff
@@ -33,6 +33,7 @@
 import java.util.concurrent.atomic.AtomicInteger;
 
 import static org.junit.Assert.*;
```

Comment:
Please, remember that one of the AC points is to replace JUnit assertions with AssertJ